rust/ql/lib/codeql/rust/frameworks/stdlib/lang-core.model.yml

```diff
@@ -3,4 +3,17 @@ extensions:
       pack: codeql/rust-all
       extensible: summaryModel
     data:
+      # Option
       - ["lang:core", "<crate::option::Option>::unwrap", "Argument[self].Variant[crate::option::Option::Some(0)]", "ReturnValue", "value", "manual"]
+      - ["lang:core", "<crate::option::Option>::unwrap", "Argument[self]", "ReturnValue", "taint", "manual"]
```

Comment:
All these `taint` models should not be needed after altering the summary above.

Reply:
I've moved one of them. But some of our sources specify taint on the entire `Result`, so I think I'd be fine to keep the others until that is no longer the case.

Reply:
I'd rather that we remove these lines, and not have flow for now, we should soon be able to have it once https://github.com/github/codeql/pull/18298 lands. Otherwise I fear we forget to remove these lines.